_posts/2019-11-18-pandera-dataframe-validation.md

```diff
@@ -13,6 +13,7 @@ categories:
   - pandas
   - data-validation
   - dataframes
+  - highlight
```

Comment:
this allows us to highlight a blog post on the home page. Most recent isn't always ideal because for isntance one recent post is - we are at agu... that's just not relevant.